plugins/zoom/icon_zoom/tasks/monitor_sign_in_out_activity/task.py

```diff
@@ -83,6 +83,7 @@ def run(self, params={}, state={}, custom_config={}):
     def loop(self, state: Dict[str, Any], custom_config: Dict[str, Any]):  # noqa: C901
         now = self._format_datetime_for_zoom(dt=self._get_datetime_now())
 
+        cutoff = custom_config.get("cutoff", 24)
```

Comment:
minor, we could include a variable at the top with 24 as the default?

Reply:
this has been updated to use DEFAULT_CUTOFF_HOURS